### DISCUSSION TITLE
Kill process when ondemand feed is closed

### DISCUSSION THREAD

**Discussion by augustodinizl** [bluenviron/mediamtx#394] [Questions] [answered]
Hello, good evening.

It is possible to kill the rtsp-simple-server process when the on-demand stream is closed? Like hitting ctrl+c

i'm trying to make it a "event driven" solution for my actual setup described here:
https://github.com/aler9/rtsp-simple-server/discussions/379#discussioncomment-768936

or maybe is possible to end the pipe when the ondemand video is disconnected

TIA

**Comment by aler9** [comment, marked as answer]
i replied in the previous thread
